test/orchard/spec_test.clj

```diff
@@ -20,3 +20,35 @@
                     (clojure.core/fn [%] (clojure.core/< (:start %) (:end %))))
              :ret (clojure.core/fn [%] (clojure.core/> (:start %) (:end %)))
              :fn nil)))))
+
+(def spec-available? (or (resolve (symbol "clojure.spec.alpha" "get-spec"))
```

Comment:
Now I think there's also alpha2.

Reply:
As far as I can tell, spec-alpha2 isn't even at pre-release stage and subject to heavy changes... I don't think we should use that API yet.